src/Illuminate/Contracts/Process/ProcessResult.php

```diff
@@ -39,6 +39,13 @@ public function exitCode();
      */
     public function output();
 
+    /**
+     * Get the standard output of the process as Stringable.
+     *
+     * @return \Illuminate\Support\Stringable
+     */
+    public function stringable();
+
```

Comment:
As this is not an internal interface, other libraries or apps could extends this class (highly unlikely though). So this would be a change for Laravel 12 in my opinion.